snippets/sortedIndex.md

```diff
@@ -0,0 +1,18 @@
+### sortedIndex
+
+Returns the lowest index at which value should be inserted into array in order to maintain its sort order
+
+```js
+const sortedIndex = (arr,n) => {
+    arr[0] > arr[1] ? (anarray = arr.reverse(),isReversed = true) : (anarray = arr,isReversed = false);  
```

Comment:
`anarray` and `isReversed` become global variables here

Reply:
Oh yeah let me fix it